src/Symfony/Bridge/PhpUnit/Legacy/ConstraintTraitForV8.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+/**
+ * @internal
+ */
+trait ConstraintTraitForV8
```

Comment:
This trait is 100% identical to `ConstraintTraitForV7` is it needed?

Reply:
#38679 adds the `exporter` method to `ConstraintTraitForV7`, which shouldn't be in this one, hence the separate trait.